### ISSUE TITLE
Which games should we have?

### ISSUE THREAD

**Issue by iteles** [dwyl/home#118]
I'm very keen to get as many collaborative board games as possible, but also games that are fun for larger groups of people.

Settlers of Catan (with the 5-6 player explansion) and Absolute Balderdash are non-negotiables 😉 

Which games would you recommend?

**Comment by iteles** [comment]
I'm so tempted by this, it looks like a lot of fun:
https://www.kickstarter.com/projects/gapclosergames/rival-restaurants-back-for-seconds/faqs
![image](https://user-images.githubusercontent.com/4185328/75909036-2e403100-5e43-11ea-8747-950deabd2d56.png)

**Comment by iteles** [comment]
https://irrationalgame.com/

![image](https://user-images.githubusercontent.com/4185328/76428508-a9569980-63a5-11ea-8ba2-fc4f7a26efac.png)

**Comment by iteles** [comment]
Tiny Epic Pirates looks like a lot of fun and is a 45 minute game:
https://www.kickstarter.com/projects/coe/tiny-epic-pirates/description

![image](https://user-images.githubusercontent.com/4185328/83908122-7599ce80-a75e-11ea-9013-0886899ef585.png)


Always a fan of games that offer multiple combinations.

**Comment by nelsonic** [comment]
@iteles any game where you get to be a _pirate_ has my vote. 🏴‍☠️ 😉

**Comment by iteles** [comment]
The Irrational Game crew has just done something brilliant. 
They sent out an email yesterday with a 20% off voucher and noting they would send another one out today - the voucher could be a higher discount or a lower one.

Today I got this:
![image](https://user-images.githubusercontent.com/4185328/88195863-6b616000-cc38-11ea-80b9-22397dc2e052.png)

😂

**Comment by nelsonic** [comment]
@iteles what are you thinking? Order it today from a diff email address and cancel it if the discount is bigger tomorrow?

**Comment by iteles** [comment]
No, I wouldn't do that! I was just noting the genius of it and how in keeping with the theme it is 😄  

I really like the idea of the game but it's very much a one-time play type of game because once you've played it once you've basically memorised the cards. I don't find much interest in those games, you get more out of reading a book (though this works for people that learn in a different way).

A very low priority expenditure for now 😊

**Comment by nelsonic** [comment]
@iteles I respectfully disagree. If I can play a game with other people, I'm _much_ more likely to _remember_ the knowledge than if I read it in a book without any practical application. 💭 
I think $39 (_even without the 20-25% discount_) is really good value for learning something useful.
Even if we only play it _once_ by ourselves, other people who visit the house will play it too. 
And if it's more _fun_, _social_ and _interactive_ than sitting alone reading a book, then I'm all for it! 

### “_Learning is experience. Everything else is just information_.” ~ Albert Einstein
